qhana_plugin_runner/db/models/tasks.py

```diff
@@ -101,12 +103,12 @@ class ProcessingTask:
         data (dict): dict-like key-value store for additional lightweight task data. New elements of type :class:`TaskData` can be added or retrieved as in a dict using ``key`` as key.
         steps (OrderingList[Step]): ordered list of steps of type :class:`Step`. Index ``number`` automatically increases when new elements are appended. Note: only use :meth:`add_next_step` to add a new step. Steps must not be deleted.
         current_step (int): index of last added step.
-        progress_value (int): progress value. ``None`` by default.
+        progress_value (float): current progress value. ``None`` by default.
```

Comment:
start and target should also be floats (target can be used as a time estimate then and start if the estimate should be displayed as a countdown).